main.go

```diff
@@ -68,7 +68,7 @@ var (
 func init() {
 	flag.BoolVar(&printVersionInfo, "version", false, "Print Elastic Package Registry version")
 	flag.StringVar(&address, "address", "localhost:8080", "Address of the package-registry service.")
-	flag.StringVar(&metricsAddress, "metrics-address", "", "Address to expose the Prometheus metrics.")
+	flag.StringVar(&metricsAddress, "metrics-address", "", "Address to expose the Prometheus metrics (experimental). ")
```

Comment:
Just a minor change here to add this Prometheus metric endpoint is subject to changes.
Do you think it is worthy to add it ?

Reply:
Sounds good to me, we may redefine something else while we start using it.